crates/sui-e2e-tests/tests/reconfiguration_tests.rs

```diff
@@ -609,11 +642,168 @@ async fn test_reconfig_with_committee_change_basic() {
+    let (total_stake, initial_validators, low_stake_grace_period) =
+        test_cluster.fullnode_handle.sui_node.with(|node| {
+            let system_state = node
+                .state()
+                .get_sui_system_state_object_for_testing()
+                .unwrap()
+                .into_sui_system_state_summary();
+
+            (
+                system_state.total_stake,
+                system_state
+                    .active_validators
+                    .iter()
+                    .map(|v| v.sui_address)
+                    .collect::<Vec<_>>(),
+                system_state.validator_low_stake_grace_period,
+            )
+        });
+
+    // Setting voting power to roughly ~ .20% of the total voting power.
+    // This allows us to achieve the following by halving:
+    // 0. .20% > VALIDATOR_MIN_POWER_PHASE_1
+    // 1. .10% > VALIDATOR_LOW_POWER_PHASE_1
+    // 2. .5%  > VALIDATOR_VERY_LOW_POWER_PHASE_1
+    let min_join_stake = total_stake * 20 / 10_000;
+    let default_stake = total_stake / initial_num_validators as u64;
+
+    execute_add_validator_transactions(&mut test_cluster, &new_validator, Some(min_join_stake))
+        .await;
+
+    test_cluster.trigger_reconfiguration().await;
+
+    // Check that a new validator has joined the committee.
+    test_cluster.fullnode_handle.sui_node.with(|node| {
+        assert_eq!(
+            node.state()
+                .epoch_store_for_testing()
+                .committee()
+                .num_members(),
+            initial_num_validators + 1
+        );
+    });
+
+    // Double the stake of every other validator, stake just as much as they had.
+    execute_add_stake_transaction(
+        &mut test_cluster,
+        initial_validators
+            .iter()
+            .map(|address| (*address, default_stake))
+            .collect::<Vec<_>>(),
+    )
+    .await;
+
+    test_cluster.trigger_reconfiguration().await;
+
+    // Find the candidate in the `active_validators` set, and check that the
+    // voting power has decreased. Panics if the candidate is not found.
+    test_cluster.fullnode_handle.sui_node.with(|node| {
+        let system_state = node
+            .state()
+            .get_sui_system_state_object_for_testing()
+            .unwrap()
+            .into_sui_system_state_summary();
+
+        let candidate = system_state
+            .active_validators
+            .iter()
+            .find(|v| v.sui_address == address);
+
+        assert!(candidate.is_some());
+        let candidate = candidate.unwrap();
+
+        // Check that the validator voting power has decreased just below the
+        // "min" threshold but not below the "low" threshold.
+        // Yet the candidate is not at risk.
+        assert!(candidate.voting_power < VALIDATOR_MIN_POWER_PHASE_1);
+        assert!(candidate.voting_power > VALIDATOR_LOW_POWER_PHASE_1);
+        assert_eq!(system_state.at_risk_validators.len(), 0);
```

Comment:
I do not really understand how any of this works, but what about the other phases? Should we be testing those as well? Are we somehow?

Reply:
The base behavior is covered in Move tests and doesn't change at all with future phases. So to shorten simtests, I suggest we don't follow this route?